app/controllers/avatars_controller.rb

```diff
@@ -0,0 +1,60 @@
+class AvatarsController < ApplicationController
+  before_action :find_user
+  before_action :set_size
+  before_action :set_theme
+
+  def show
+    gravatar_url = @user.gravatar_url(size: @size, default: "404", secure: true)
+
+    resp = Faraday.get(gravatar_url, nil,
```

Comment:
What's Faraday's timeout here? If Gravatar's servers are being particularly spotty, we wouldn't want to be waiting around forever for an answer. Should we put a timeout of something like 5 seconds in?